lib/lita/handlers/cmd.rb

```diff
@@ -11,6 +11,7 @@ class Cmd < Handler
       config :stdout_prefix, default: ""
       config :stderr_prefix, default: "ERROR: "
       config :command_prefix, default: "cmd "
+      config :ignore_script, default: Regexp.union(/~/, /#/)
```

Comment:
This is `ignore_script`, but everywhere else it is referenced as `ignore_scripts` (plural). Typo?